misk/src/test/kotlin/misk/web/ssl/Http2ConnectivityTest.kt

```diff
@@ -241,6 +279,30 @@ class Http2ConnectivityTest {
     }
   }
 
+  class LockInterceptor constructor(val queue: ArrayBlockingQueue<Int>) : NetworkInterceptor {
```

Comment:
don't remember exactly but do we still need this?

Reply:
yeah, this so the test can know when they can assert the side effect on the server side after the timeout.